hasher-matcher-actioner/src/OpenMediaMatch/blueprints/curation.py

```diff
@@ -212,6 +212,19 @@ def _bank_add_signals(
     }
 
 
+@bp.route("/bank/<bank_name>/content/<content_id>", methods=["DELETE"])
+def bank_delete_content(bank_name: str, content_id: int):
+    """
+    Remove a signal from a bank.
+    """
+    storage = persistence.get_storage()
+    bank = storage.get_bank(bank_name)
+    if not bank:
+        abort(404, f"bank '{bank_name}' not found")
```

Comment:
blocking: One thing we might need to do here is protect import banks, whose content are managed by the exchange config. If you were planning on using this to move content between exchange configs, then we'll be working on this exact problem for the Child safety hackathon!

Reply:
ah this is specifically for non imported banks.

I updated the `bank_remove_content` method but let me know if you want me to split this out into a different method